docs/04-api-reference/front-commerce-remix/index.mdx

```diff
@@ -91,33 +124,75 @@ Instead of the standard remix `SerializeObject` type:
 SerializeObject<UndefinedToOptional<{ foo: "bar"; date: string }>>;
 ```
 
-## Front-Commerce Context
+### `useApiFetcher`
 
-To ensure the Front-Commerce Context is available in your loaders and actions,
-you need to add the `createFrontCommerceContext` to your server via the
-[`getLoadContext`](https://remix.run/docs/en/2.0.1/route/loader#context), to do
-this, update your
-[`server.ts`](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/b99f0475ce494591a54fdc2b904e956a083e9b51/skeleton/server.ts)
+The `useApiFetcher` is an optimised fetcher which allows to deduplicate client
+side calls the the same `/api` endpoint. You can replace a load of `useFetcher`
```

Comment:
> You can replace a load of `useFetcher` calls

I'm not sure what this means, did you meant "a lot of `useFetcher` calls"?

Reply:
I will rephrase, it, I make reference to the `load` method returned from the `useFetcher` hook, but after re-reading I notice it's not clear.

Reply:
Ooh makes sense. Thanks !